sass/_Omnibox.scss

```diff
@@ -5,18 +5,35 @@
   z-index: 0;
 }
 .CoveoOmnibox.magic-box {
-  .coveo-omnibox-hightlight {
```

Comment:
Renaming the CSS classes, even if to correct a typo, would be a breaking change unfortunately. Some of our clients are probably counting on the old name being present in their CSS rules.